aldryn_forms/boilerplates/bootstrap3/templates/aldryn_forms/fields/filefield.html

```diff
@@ -0,0 +1,20 @@
+{% load aldryn_forms_tags i18n %}
+
+{% if field %}
+    <div class="form-group{% if field.field.required %} form-required{% endif %}{% if field.errors %} has-error{% endif %}">
+        {% block form_field_label %}
+            <label>{{ field.label }}{% if field.field.required %}<abbr title="{% trans "Required field" %}">*</abbr>{% endif %}</label>
```

Comment:
the label needs to be always associated with the input through ``<label for="field-input-1234">` and `<input id="field-input-1234">` for all cases